live-examples/js-examples/intl/intl-supportedvaluesof.js

```diff
@@ -0,0 +1,14 @@
+console.log(Intl.supportedValuesOf('calendar'));
+console.log(Intl.supportedValuesOf('collation'));
+console.log(Intl.supportedValuesOf('currency'));
+console.log(Intl.supportedValuesOf('numberingSystem'));
+console.log(Intl.supportedValuesOf('timeZone'));
+console.log(Intl.supportedValuesOf('unit'));
+// expected output: Array ['key'] (for each key)
```

Comment:
I know this format is non standard - usually you would have the comment line after each call. However ALL of these are the same so the extra line just pads things out. What this does though is mean that I can show the entire use of the API within the allocated space. Which I see as a good thing.